metrics/http.go

```diff
@@ -29,6 +29,13 @@ var (
 		},
 		[]string{"api"},
 	)
+	Version = Factory.NewCounterVec(
+		prometheus.CounterOpts{
```

Comment:
should it be counter or gauge? (few suggestions only stated gauge is better for one-off info). redis also uses gauge to export its version/deployment info as metrics

Reply:
Maybe! I'll switch it, since I have no idea